src/content/ja/2019/fonts.md

```diff
@@ -0,0 +1,651 @@
+        <td class="numeric">6.6%</td>
+      </tr>
+      <tr>
+        <td>maxcdn.bootstrapcdn.com</td>
+        <td class="numeric">1.8%</td>
+        <td class="numeric">2.0%</td>
+      </tr>
+      <tr>
+        <td>use.fontawesome.com</td>
+        <td class="numeric">1.1%</td>
+        <td class="numeric">1.2%</td>
+      </tr>
+      <tr>
+        <td>static.parastorage.com</td>
+        <td class="numeric">0.8%</td>
+        <td class="numeric">1.2%</td>
+      </tr>
+      <tr>
+        <td>fonts.shopifycdn.com</td>
+        <td class="numeric">0.6%</td>
+        <td class="numeric">0.6%</td>
+      </tr>
+      <tr>
+        <td>cdn.shopify.com</td>
+        <td class="numeric">0.5%</td>
+        <td class="numeric">0.5%</td>
+      </tr>
+      <tr>
+        <td>cdnjs.cloudflare.com</td>
+        <td class="numeric">0.4%</td>
+        <td class="numeric">0.5%</td>
+      </tr>
+      <tr>
+        <td>use.typekit.com</td>
+        <td class="numeric">0.4%</td>
+        <td class="numeric">0.4%</td>
+      </tr>
+      <tr>
+        <td>netdna.bootstrapcdn.com</td>
+        <td class="numeric">0.3%</td>
+        <td class="numeric">0.4%</td>
+      </tr>
+      <tr>
+        <td>fast.fonts.net</td>
+        <td class="numeric">0.3%</td>
+        <td class="numeric">0.3%</td>
+      </tr>
+      <tr>
+        <td>static.dealer.com</td>
+        <td class="numeric">0.2%</td>
+        <td class="numeric">0.2%</td>
+      </tr>
+      <tr>
+        <td>themes.googleusercontent.com</td>
+        <td class="numeric">0.2%</td>
+        <td class="numeric">0.2%</td>
+      </tr>
+      <tr>
+        <td>static-v.tawk.to</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.3%</td>
+      </tr>
+      <tr>
+        <td>stc.utdstc.com</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.2%</td>
+      </tr>
+      <tr>
+        <td>cdn.jsdelivr.net</td>
+        <td class="numeric">0.2%</td>
+        <td class="numeric">0.2%</td>
+      </tr>
+      <tr>
+        <td>kit-free.fontawesome.com</td>
+        <td class="numeric">0.2%</td>
+        <td class="numeric">0.2%</td>
+      </tr>
+      <tr>
+        <td>open.scdn.co</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.1%</td>
+      </tr>
+      <tr>
+        <td>assets.squarespace.com</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.1%</td>
+      </tr>
+      <tr>
+        <td>fonts.jimstatic.com</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.2%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>図2. リクエストの割合による上位20のフォントホスト。</figcaption>
+</figure>
+
+ここでのGoogle Fontsの優位性は、同時に驚くべきことであると同時に意外性のないものであった。期待していたという点では予想外でしたが、サービスの圧倒的な人気の高さには驚きました。フォントリクエストの75％というのは驚異的だ。TypeKitは一桁台の遠い2位で、Bootstrapライブラリがさらに遠い3位を占めていました。
+
+<figure>
+  <div class="big-number">29%</div>
+  <figcaption>図3. ドキュメント内にGoogle Fontsスタイルシートのリンクを含むページの割合 <code>&lt;head></code>。</figcaption>
```

Comment:
Looks wrong in English. Will fix in other chapters, but let's fix Japanese here.
```suggestion
  <figcaption>図3. ドキュメント内にGoogle Fontsスタイルシートのリンクを含むページの割合 <code>&lt;head&gt;</code>。</figcaption>
```

Reply:
Fixed this!